chapter_02/lewis/exercise_1.pl

```diff
@@ -0,0 +1,18 @@
+###
+# (C) 2017 Lewis Maitland
+# O'Reily Excersises
+###
+
+use strict;
+use warnings;
+use Math::Trig ':pi';
+
+my $radius = $#ARGV == 0 ? $ARGV[0] : 12.5;

```

Comment:
$#ARGV works, but probably not for the reason you think it does. If you want to check how many elements are in an array, use ``scalar``.